File: Debug/Python/1.py
# Find whehter the given string is palindrome or not

def is_palindrome(string):
    clean_string = ' '.join(char for char in string)
    clean_string == clean_string[::-1]
    return clean_string

inputs = ['palindrome', 'ala', 'helle']

for input_string in inputs:
    if is_palindrome(input_string):
        print(f"'{input_string}' is a palindrome.")
    else:
        print(f"'{input_string}' is not a palindrome.")

File: Debug/C++/1.cpp
// Find whehter the given string is palindrome or not

#include <iostream>
#include <string>
#include <algorithm>

using namespace std;

string isPalindrome(string str)
{
    string cleanString = str;
    reverse(cleanString.end(), cleanString.begin());
    return cleanString == str ? " is a palindrome." : " is not a palindrome.";
}

int main()
{
    string inputs[] = {"palindrome", "ala", "helle"};

    for (const string &inputString : inputs)
    {
        cout << "'" << inputString << "'" << isPalindrome(inputString) << endl;
    }

    return 0;
}
